langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -93,6 +126,12 @@ public isolated function codePointCompare(string str1, string str2) returns int
 
 # Joins zero or more strings together with a separator.
 #
+# ```ballerina
+# string introString = string:'join(" ", "Ballerina", "is", "a", "statically", "typed,", "concurrent", "programming", "language");
+#
+# string student = string:'join(",", "John", "23", "USA", "Computer Science", "Swimmer");
```

Comment:
Do we need this example. It is same as first one ryt?. Instead of that shall we use a array of strings.

Reply:
Updated.